src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -159,52 +81,74 @@ const ChatItemMyTextV2 = ({
     );
   }, []);
 
+  const contextMenuActions = [
+    {title: 'Reply', systemIcon: 'arrow.turn.up.left'},
+    {title: 'Copy Message', systemIcon: 'square.on.square'},
+    {title: 'Delete Message', systemIcon: 'trash', destructive: true}
+  ];
+
   const renderAvatar = React.useCallback(() => {
-    if (isContinuous) return <View style={styles.avatar} />;
+    if (!isShowUserInfo) return <View style={styles.avatar} />;
     return <View style={styles.ml8}>{avatar}</View>;
   }, []);
 
-  const handleTextContainerStyle = () => {
-    if (chatType === SIGNED) {
-      return [styles.containerSigned, styles.textContainer];
-    }
-    return [styles.containerAnon, styles.textContainer];
+  const onSwipeToReply = (direction) => {
+    if (direction === 'right') return;
+    if (swipeableRef.current) swipeableRef.current?.close();
+    setReplyPreview({
+      username,
+      time,
+      message,
+      messageId: data?.id,
+      chatType,
+      messageType: 'regular'
+    });
   };
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You have defined context menu actions, but there are no handlers for these actions. You should add handlers for each action to perform the corresponding operations when a user selects an action from the context menu.

```diff
  const contextMenuActions = [
    {
      title: 'Reply',
      systemIcon: 'arrow.turn.up.left',
+     onSelect: () => handleReply()
    },
    {
      title: 'Copy Message',
      systemIcon: 'square.on.square',
+     onSelect: () => handleCopyMessage()
    },
    {
      title: 'Delete Message',
      systemIcon: 'trash',
      destructive: true,
+     onSelect: () => handleDeleteMessage()
    }
  ];
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->